src/shared/components/news.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component } from "inferno";
+import { Helmet } from "inferno-helmet";
+import { i18n } from "../i18next";
+import { news_md } from "../translations/news";
+import { isBrowser, mdToHtml } from "../utils";
+
+const title = i18n.t("news");
+
+export class News extends Component<any, any> {
+  constructor(props: any, context: any) {
+    super(props, context);
+  }
+
+  componentDidMount() {
+    if (isBrowser()) {
+      window.scrollTo(0, 0);
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <Helmet title={title}>
+          <meta property={"title"} content={title} />
+        </Helmet>
+        <div class="container">
+          <h1>{title}</h1>
+          <div dangerouslySetInnerHTML={mdToHtml(news_md)} />
```

Comment:
We could potentially add lots of files here, but I kind of like the running log style where you can see all the updates on one page.

Reply:
That would require some extra complication, because now there'd have to be another page / separate sidebar where you select which file / news post you want to view.

Reply:
Yes there should be one overview page with headlines, and a page that renders the complete post. Shoving all of these long posts in a single page would make it completely unusable.